autopilot/hostscore.go

```diff
@@ -237,13 +245,22 @@ func uptimeScore(h api.Host) float64 {
 	return math.Pow(ratio, 200*math.Min(1-ratio, 0.30))
 }
 
-func versionScore(settings rhpv2.HostSettings) float64 {
+func versionScore(settings rhpv2.HostSettings, minVersion string) float64 {
+	if minVersion == "" {
+		minVersion = minProtocolVersion
+	}
 	versions := []struct {
 		version string
 		penalty float64
 	}{
-		{"1.6.0", 0.99},
-		{"1.5.9", 0.00},
+		// latest protocol version
+		{"1.6.0", 0.10},
```

Comment:
This is not for 1.6.0 hosts but for <1.6.0 hosts. So that we don't immediately drop 1.5.9 hosts but do have a higher chance of picking 1.6.0 if we need to replace a host.